### DISCUSSION TITLE
Documentation to remove it when installed from curl script.

### DISCUSSION THREAD

**Discussion by nithish-c** [starship/starship#5651] [Ideas]
I looked through everywhere and have not found an easy way to remove starship when installed from one-line curl command. If this issue is already discussed and found a solution, please direct me to it. 

While I could easily use cargo, or homebrew for macs, I also think it is important to have a proper documentation or a script to uninstall to revert back all changes the install script does. 

Even if I do get answers for this here in issues, I hope someone adds a section for it in documentation too. For others to find it easily.

**Comment by mickimnet** [comment]
Hello nithish-c,

the FAQ entry, [How do I uninstall Starship?](https://starship.rs/faq/#how-do-i-uninstall-starship), doesn't help you?

Take care
Mick
